go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	golang.org/x/sync v0.16.0
 	golang.org/x/term v0.34.0
 	gopkg.in/yaml.v3 v3.0.1
+	mvdan.cc/sh/moreinterp v0.0.0-20250807215248-5a1a658912aa
 	mvdan.cc/sh/v3 v3.12.0
```

Comment:
I'm not sure what Task's guideline is on referencing unreleased libraries. Do we need to wait until it's officially released (e.g., tagged on GitHub) ?

Reply:
Oh! So we could just use the latest version to fix the issue that’s impacting us?
I am talking about https://github.com/go-task/task/issues/2355 that has been fixed in the upstream but not released yet

Reply:
If we're super keen to get a fix out and it passes all our tests, then I'm not against it. Let's get @andreynering's thoughts too though.

Reply:
Thanks a lot for the explaination! 
Fine to me!